google-chart.html

```diff
@@ -469,6 +476,33 @@
         .then(function(dataView) {
           this._dataView = dataView;
         }.bind(this));
+    },
+
+    /**
+     * Queries global document head for google charts link#load-css-* and clones
+     * them into the local root's div#styles element for shadow dom support.
+     */
+    _localizeGlobalStylesheets: function() {
```

Comment:
I think my version in #204 handles some edge cases a little better.
https://github.com/GoogleWebComponents/google-chart/pull/204/files#diff-d0b4f475ae2630d8efc01169d1a41af6R80

Reply:
I merged this on behalf of an internal request from a blocked user. I'd 100% be willing to review your import CL over this implementation.